accounts/views.py

```diff
@@ -91,8 +93,14 @@ def form_valid(self, form):
                 slack_user = lookup_user(self.object.email)
                 if slack_user:
                     user_add(settings.SLACK_TARGET_EXEC, slack_user)
-
-            # TODO: Add new active members to the active channel in Slack (and remove inactive / alumni etc.)
+            elif active_group not in oldgroups:
```

Comment:
I don’t think this logic will work the way you want it to. Adding or removing users from the active channel should probably be separate from adding or removing them from exec.